src/phases.ts

```diff
@@ -2340,38 +2338,51 @@ export class TurnStartPhase extends FieldPhase {
         const aMove = allMoves[aCommand.move!.move];//TODO: is the bang correct here?
         const bMove = allMoves[bCommand!.move!.move];//TODO: is the bang correct here?
 
+        // The game now considers priority and applies the relevant move and ability attributes
         const aPriority = new Utils.IntegerHolder(aMove.priority);
         const bPriority = new Utils.IntegerHolder(bMove.priority);
+        const isSameBracket = aPriority === bPriority;
```

Comment:
```suggestion
        const isSameBracket = aPriority.value === bPriority.value;
```
Also, why is this set before attributes that can change priority are applied?

Reply:
This checks if the **original** priority of the moves are the same.